src/components/tabs/chapters/ChapterEntry.vue

```diff
@@ -139,6 +150,9 @@
         },
 
         onChapterClick: function ({ dispatch, actions }, event) {
+          if (event.target.classList.contains('info-link')) {
```

Comment:
`onChapterClick` is something what we call "advanced interactions". It is for users that are explicitly pressing the `alt` key to select a chapter playtime. Because this is a very conscious interaction I wold not prevent it even if the link was target of the interaction.